tests/modules/test_urllib3.py

```diff
@@ -0,0 +1,59 @@
+import urllib3
+import epsagon.wrappers.python_function
+from epsagon.trace import trace_factory
+import epsagon.runners.python_function
+import epsagon.constants
+
+
+TEST_URL = 'https://google.com/'
+
+def setup_function(func):
+    trace_factory.use_single_trace = True
+    epsagon.constants.COLD_START = True
+
+def test_no_data_capture_on_preload_content_false(trace_transport):
+    @epsagon.wrappers.python_function.python_wrapper
+    def wrapped_function():
+        headers = {
+            'Content-Type': 'application/json'
+        }
+
+        http = urllib3.PoolManager()
+        conn = http.connection_from_url(TEST_URL)
+        urllib_response = conn.urlopen(
+            method='POST',
```

Comment:
`GET`?

Reply:
maybe better to test both cause the body may be handled differently

Reply:
Fixed